psa-crypto-sys/build.rs

```diff
@@ -171,6 +171,7 @@ mod common {
             .clang_arg(format!("-I{}", mbed_include_dir))
             .header("src/c/shim.h")
             .blocklist_type("max_align_t")
+            .blocklist_type("psa_crypto_driver_pake_step_t")
```

Comment:
Do we know why bindgen generated this?

Reply:
Isn't it because it gets defined in files like this one `https://github.com/Mbed-TLS/mbedtls/blob/2107feb7a6dfa29b3121d7412defc9772c053aca/include/psa/crypto_extra.h#L1804` ?

Reply:
Meaning, there should be a version of mbedtls that is defining that constant in the crypto.h file

Reply:
Please see: [crypto.h#L4833](https://github.com/Mbed-TLS/mbedtls/blob/development/include/psa/crypto.h#L4833)

In here, `crypto.h` is including `crypto_extra.h`

Reply:
Sorry for not being clear, what I meant is why is this getting generated now and not earlier. 

https://github.com/Mbed-TLS/mbedtls/commit/251e86ae3f19c3866650b517f61a80cd881f44a3#diff-8fe19c51d861f48010b3609673ee1dd94bb3eef20af920af31b36fc0dcb124c8L2005

Reply:
`psa_pake_driver_step_t` is renamed to `psa_crypto_driver_pake_step_t`

Reply:
I think I was able to solve this now